### DISCUSSION TITLE
How can i login into single account under my organization

### DISCUSSION THREAD

**Discussion by ztitin** [postalserver/postal#1856] [Help with using Postal]
i build a organization domain is example.com and i receiving a mail for the account test@example.com, i can view the email in the admin page, but my question is how can i login into the test@example.com account and view the receive mail.

**Comment by willpower232** [comment]
Postal does not function like a traditional mailserver with mailboxes, its primary purpose is to send emails from software that can't usually send emails properly like website CMS's or custom internal software.
